modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RedirectButton/RedirectButton.scss

```diff
@@ -0,0 +1,13 @@
+.redirect-button {
+	margin-top: 100px;
+
+	.sheet-text {
+		font-size: 0.875rem;
+		margin-bottom: 0.75rem;
+	}
+
+	label {
```

Comment:
@rodrigopaulino, can we sort it alphabetically?

Reply:
@natocesarrego, I'm not sure what you want to sort here? Are you talking about `.sheet-text` and `label`?

Reply:
Sorry, @rodrigopaulino. I meant sort the CSS properties / classes `margin-top`, `.sheet-text` and `label`. :wink:

Reply:
No worries, @natocesarrego. Is there any examples of sorting properties, class selectors and tag selectors like that? I understand doing it within each type of asset... but I struggle to see the value of placing, let's say, a property from the parent CSS class in the middle of its body and another one at end. Take our Java internal convention for example: we group class' attributes and methods separate from each other and then we sort within those groups alphabetically. Does it make sense to you?

Reply:
I think I got your point :slightly_smiling_face: 

Let's move on and ask a front-end developer about this CSS convention. :wink:

Reply:
Do you want me to do it? :)

Reply:
No need, @rodrigopaulino. :wink: Let's move on. :slightly_smiling_face: